ChangeLog.d/remove_allow_sha1_in_certificates

```diff
@@ -0,0 +1,15 @@
+Removals
+   * Remove optional SHA-1 in the default TLS configuration for certificate
+     signing. This feature was ment to be available only temporarily.
+     Users are expected to use SHA-2 instead, since SHA-1 is currently
+     considered a security risk.
+     If needed, SHA-1 cerificate can still be used by providing custom
```

Comment:
I think this level of detail belongs in a separate place (a knowledge base entry): it's too much information for the changelog. But I'm ok with having it here now and moving it later (before the 3.0 release!).

Reply:
@daverodgman suggested this is a bit too niche to include in mbedTLS docs.